src/app.go

```diff
@@ -7,48 +7,42 @@ import (
 	aliasapp "github.com/consensys/quorum-key-manager/src/aliases/app"
 	"github.com/consensys/quorum-key-manager/src/auth"
 	"github.com/consensys/quorum-key-manager/src/infra/log"
+	manifests "github.com/consensys/quorum-key-manager/src/infra/manifests/filesystem"
 	"github.com/consensys/quorum-key-manager/src/infra/postgres/client"
-	"github.com/consensys/quorum-key-manager/src/manifests"
-	manifestsmanager "github.com/consensys/quorum-key-manager/src/manifests/manager"
 	"github.com/consensys/quorum-key-manager/src/nodes"
 	stores "github.com/consensys/quorum-key-manager/src/stores/app"
 	"github.com/justinas/alice"
 )
 
 type Config struct {
-	HTTP      *server.Config
-	Logger    *log.Config
-	Manifests *manifestsmanager.Config
-	Postgres  *client.Config
-	Auth      *auth.Config
+	HTTP     *server.Config
+	Logger   *log.Config
+	Postgres *client.Config
+	Auth     *auth.Config
+	Manifest *manifests.Config
 }
 
 func New(cfg *Config, logger log.Logger) (*app.App, error) {
 	// Create app
 	a := app.New(&app.Config{HTTP: cfg.HTTP}, logger.WithComponent("app"))
 
 	// Register Service Configuration
-	err := a.RegisterServiceConfig(cfg.Manifests)
+	err := a.RegisterServiceConfig(cfg.Auth)
```

Comment:
IRCC auth config should also include `cfg.manifest` as Roles are loaded from manifest files